lib/services/github_issues.rb

```diff
@@ -256,6 +256,8 @@ def issue_body(resource)
     if resource.description.attachments.present?
       issue_body_parts << attachments_in_body(resource.description.attachments)
     end
+    issue_body_parts << "Created from Aha! #{resource.url}" if resource.respond_to?(:url)
```

Comment:
How does this work with webhooks? If the description is updated in Github will it appear in Aha! with this link? Will the link be added a second time when it is send back to Github?